CallsheetLocalizations/es-MX.xcloc/Localized Contents/es-MX.xliff

```diff
@@ -744,6 +746,7 @@
       </trans-unit>
       <trans-unit id="Light" xml:space="preserve">
         <source>Light</source>
+        <target state="translated">Luz</target>
```

Comment:
Apple docs call this "Claro" instead of "Luz" which I think is more accurate.

Reply:
85139fb4c4ef5866a3af74013c4aa8130e08481b